packages/flutter/test/material/switch_list_tile_test.dart

```diff
@@ -278,7 +276,7 @@ void main() {
     );
 
     await tester.pump();
-    expect(Focus.of(childKey.currentContext, nullOk: true).hasPrimaryFocus, isTrue);
+    expect(Focus.of(childKey.currentContext!, nullOk: true)!.hasPrimaryFocus, isTrue);
```

Comment:
The combination of `nullOk: true` and `!` is strange. Looks like really null is not OK here.

Reply:
Good catch. Fixed.